src/FactSystem/FactControls/FactStepper.qml

```diff
@@ -0,0 +1,130 @@
+/****************************************************************************
+ *
+ *   (c) 2009-2016 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
+
+import QtQuick 2.5
+import QtQuick.Controls 1.2
+import QtQuick.Controls.Styles 1.2
+import QtQuick.Dialogs         1.2
+
+import QGroundControl.ScreenTools   1.0
+import QGroundControl.FactSystem 1.0
+import QGroundControl.Palette 1.0
+import QGroundControl.Controls 1.0
+
+Item {
+    id: stepper
+    property bool enabled: true
+    property alias fact: factInput.fact
+    property alias showHelp: factInput.showHelp
+    property alias showUnits: factInput.showUnits
+
+    property string _valueText:          fact ? fact.valueString : "N/A"
+    property double _factValue:          Number(_valueText)
+
+    property double minimumValue:       !fact || isNaN(fact.min) ? 0 : fact.min
+    property double maximumValue:       !fact || isNaN(fact.max) ? 1 : fact.max
```

Comment:
Why all the checking for !fact? If there is no fact set then you don't really want to hide that fact since it is badly written code.